Monika After Story/game/script-topics.rpy

```diff
@@ -4847,6 +4847,38 @@ label monika_coffee:
     m 1k "Maybe if I find out how, I'll tinker with the script so I can finally drink some! Ahaha~"
     return
 
+init 5 python:
+    for key in ['clothing', 'looks', 'preferences', 'uniform']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_clothing')
+    monika_random_topics.append('monika_clothing')
+        
+label monika_clothing:
+    m 3d "I have been wondering [player], do you like the way my clothing looks."
+    m 3c "I mean, all I wore by now is my outfit for school." 
+    m 3c "So you didn't see me in something else than this like the other girls." 
+    m 4l "Not that I am jealous!" 
+    m 5m "I just like it if you enjoy looking at me."  
+    m 2r "Also school uniforms are not as comfortable as people think." 
+    m 4n "At least not for us girls." 
+    m 1q "In winter my legs are freezing."
+    m 1q "And in summer I'm sweating very fast in this blazer."
+    m 5j "On the other hand, people like them and think they look cute. Ahaha~"
+    m 1a "But what about you [player], would you like me to change my outfit sometime?"  
+    menu:
+        "Yes":
+            m 2k "Ahaha, I knew this was a good idea." 
+	        m 2j "Maybe I will find some new clothes in my code soon."
```

Comment:
Fix indents.

Reply:
quick question: To fix stuff, just edit the file again as before ?

Reply:
sry never used this kollab system before